public/app/features/alerting/unified/components/rule-editor/SelectWIthAdd.tsx

```diff
@@ -29,13 +30,15 @@ export const SelectWithAdd: FC<Props> = ({
   disabled = false,
   addLabel = '+ Add new',
   'aria-label': ariaLabel,
+  getOptionLabel,
 }) => {
   const [isCustom, setIsCustom] = useState(custom);
 
   useEffect(() => {
-    if (custom) {
-      setIsCustom(custom);
-    }
+    //not sure why do we need this if
```

Comment:
I need to be able to hide the group input once user selects another folder (group drop-down should be reset)